sale_comment_propagation/__manifest__.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Ainara Galdona - AvanzOSC
+# Copyright 2015 Oihane Crucelaegui - AvanzOSC
+# Copyright 2015 Esther Martín <esthermartin@avanzosc.es> - Avanzosc S.L.
+# Copyright 2016-2017 Lorenzo Battistini - Agile Business Group
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Comments for sale documents (order, picking and invoice)",
+    "version": "10.0.0.1.0",
```

Comment:
Follow guidelines for version numbering: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers. This migration should start on 10.0.1.0.0 and then bump according to said guidelines.